_includes/video/pb-os-nas.html

```diff
@@ -1,49 +1,54 @@
 <!DOCTYPE html>
 <html lang="en" class="html--no-js">
-
-<head>
-{% include head--common.html %}
-    <!--production version of prebid.js-->
-    <script async src="//cdn.jsdelivr.net/npm/prebid.js@latest/dist/not-for-prod/prebid.js"></script>
+  <head>
+    {% include head--common.html %} {% include prebidjs-non-prod.html %}
 
     <!--pbjs and player code -->
-  <script type="text/plain" class="optanon-category-C0003">
-	    var pbjs = pbjs || {};
-	    pbjs.que = pbjs.que || [];
+    <script>
```

Comment:
not clear why this isn't checking for CMP approval -- outstream renderers could set cookies like in-page players, right?

Reply:
This code does nothing as long the relevant js is loaded (prebid.js and player).
Only callbacks to a que are pushed